src/main/java/org/example/AccountInputService.java

```diff
@@ -37,6 +41,16 @@ public String getAccountPassword(){
         String accountPassword = sc.next();
         return accountPassword;
     }
+    public String confirmPassword(){
+        String accountPassword = getAccountPassword();
```

Comment:
why separate function, password functionality should be in one function, its one functioality

Reply:
sir getAccountPassword used in loginAccount() & confirmPassword() method that's Why i did not remove the function but now I removed it.